ci/docker-rally.sh

```diff
@@ -12,13 +15,15 @@ echo "OVS_REPO=${OVS_REPO} OVS_BRANCH=${OVS_BRANCH}"
 # A combined script to run all the things
 
 # Prepare the environment
-./prepare.sh
+./prepare.sh || FAILED=$(( $FAILED + 1 ))
 
 # Run the testsuite
-./scale-run.sh $OVS_REPO $OVS_BRANCH
+./scale-run.sh $OVS_REPO $OVS_BRANCH || FAILED=$(( $FAILED + 1 ))
```

Comment:
With the PR, shall we avoid calling exit in the other script, e.g., https://github.com/openvswitch/ovn-scale-test/blob/master/ci/scale-run.sh#L23